quarkus-opentelemetry-exporter-azure/runtime/src/main/java/io/quarkiverse/opentelemetry/exporter/azure/runtime/AzureRecorder.java

```diff
@@ -32,6 +35,43 @@ public LateBoundSpanProcessor apply(SyntheticCreationalContext<LateBoundSpanProc
         };
     }
 
+    public Function<SyntheticCreationalContext<AzureEndpointSampler>, AzureEndpointSampler> createSampler(
+            AzureExporterRuntimeConfig runtimeConfig, AzureExporterQuarkusRuntimeConfig quarkusRuntimeConfig) {
+        return new Function<>() {
+            @Override
+            public AzureEndpointSampler apply(SyntheticCreationalContext<AzureEndpointSampler> context) {
+                List<String> ingestionUrls = findIngestionUrls(runtimeConfig, quarkusRuntimeConfig);
+                List<String> statsBeatUrls = Arrays.asList("https://westeurope-5.in.applicationinsights.azure.com/",
```

Comment:
can these addresses be retrieved from somewhere?

Reply:
No. These are default end points. They are not in the connection string.